module/VuFind/src/VuFind/View/Helper/Root/AlphaBrowse.php

```diff
@@ -30,9 +30,10 @@
 namespace VuFind\View\Helper\Root;
 
 use Laminas\View\Helper\Url;
+use VuFind\Search\SearchOrigin\AbstractSearchOrigin;
 
 /**
- * Authentication view helper
+ * Authentication? view helper
```

Comment:
This was a copy-and-paste error. I've just pushed a fix up to the dev branch, so this will be sorted out when you do your next merge.